Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.11
+ADD . /usr/src/nimbess-agent
+RUN mkdir -p $GOPATH/src/github.com/nimbess/nimbess-agent
+WORKDIR $GOPATH/src/github.com/nimbess/nimbess-agent
```

Comment:
WORKDIR should create the directory so the previous RUN mkdir is not needed.

Reply:
will remove